src/main/java/org/folio/pv/controller/PasswordValidatorController.java

```diff
@@ -31,4 +35,12 @@ public PasswordValidatorController(ValidationRuleService validationRuleService)
     ValidationResult validationResult = validationRuleService.validatePasswordByRules(passwordContainer);
     return new ResponseEntity<>(validationResult, HttpStatus.OK);
   }
+
+  @ExceptionHandler(RuntimeHttpException.class)
+  public ResponseEntity<ValidationResult> handleException(RuntimeHttpException exception) {
```

Comment:
`ValidationResult` entity is used for providing the result of the password validation. For providing any exceptions, errors to client use Errors entity, it should be in classpath.

Reply:
Now using `Errors` and `Error`